ckanext/budgetviz/groupbarchart_munc/theme/public/custom.css

```diff
@@ -0,0 +1,138 @@
+.nvd3 text,
+    .nv-axislabel {
+        font-family: 'Open Sans', sans-serif;
+    }
+    
+    text {
+        font-size: 12px;
+    }
+    
+    svg {
+        display: block;
+    }
+    
+    body {
+        margin: 0 auto;
+    }
+    
+    #chart,
+    svg {
+        margin: 0 auto;
+        padding: 0px;
+        height: 400px;
+        width: 100%;
+        opacity: 0.95;
+    }
+    .infocard{
+        width:100%;
+    }
+    
+    .nvd3 .nv-axis.nv-x path.domain {
+        stroke-opacity: 1;
+    }
+    
+    svg,
+    .nv-background rect {}
+    
+    .nvd3 .nv-axis line {
+        stroke: rgba(0, 0, 0, 0.05);
+        opacity: 0.7;
+    }
+    
+    path.domain {
+        opacity: 0.8;
+    }
+    
+    .nvtooltip.with-3d-shadow,
+    .with-3d-shadow .nvtooltip {
+        border-radius: 3px;
+        font-family: 'Open Sans', sans-serif;
+    }
+    
+    .nv-point {
+        stroke-opacity: 1!important;
+        stroke-width: 1px!important;
+        fill-opacity: 0!important;
+    }
+    
+    .nv-point.hover {
+        stroke-width: 9px!important;
+        fill-opacity: .95 !important;
+        stroke-opacity: .95 !important;
+    }
+    
+    .infocard {
```

Comment:
Are we using images as background files? It would be more scalable and responsive if we use SVGs